inspec.yml

```diff
@@ -1,23 +1,28 @@
 name: cis-aws-foundations-baseline
 title: cis-aws-foundations-baseline
 maintainer: MITRE InSpec Team
-copyright: MITRE, 2018
+copyright: MITRE, 2020
 copyright_email: inspec@mitre.org
 license: Apache-2.0
 summary: 'InSpec Profile to validate the CIS AWS Foundations Benchmark v1.1.0'
 version: 1.0.5
 supports:
   - platform: aws
-
 depends:
   - name: inspec-aws
     git: https://github.com/inspec/inspec-aws.git
 
-attributes:
+inputs:
   - name: default_aws_region
     description: 'primary aws region your resources are deployed.'
     value: 'us-east-1'
 
+  - name: aws_cloudtrail_trail
```

Comment:
This input value might not be needed after the suggested update for 3.x controls

Reply:
Done!